lib/gui/app/components/image-selector/image-selector.jsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 'use strict'
+
+const React = require('react')
+const ReactDOM = require('react-dom')
+const propTypes = require('prop-types')
+const Color = require('color')
+
+const middleEllipsis = require('./../../utils/middle-ellipsis')
+
+import styled, { ThemeProvider } from 'styled-components'
```

Comment:
The mixture of `import` and `require` here is a bit weird. @jhermsmeier whats the preferred syntax in etcher?

Reply:
For consistency we've stuck with require(), as ES Modules aren't available across the board in Etcher yet (without transpiling everything), so I'd recommend going with require() for the time being.

@amdomanska keep in mind that for packages written with the ES Module style you'll have to access their `default` export directly when using require(), i.e.

```js
const styled = require('styled-components').default
```

Reply:
Aren't you already transpiling everything when you run webpack?

Reply:
Only JSX so far